doc/website/getting-started/overview.md

```diff
@@ -11,637 +10,242 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
-access the underlying ``CounterTopic`` either by getting a pointer to it via ``get`` or by using ``operator->``. In
-any case, we now can process or copy the received data and once the ``sample`` goes out of scope, the underlying
-``CounterTopic`` object is deleted as well (this happens when the temporary object returned by ``take`` is
-destroyed). This means it is only safe to hold references to the data as long as the ``sample`` exists. Should we
-need a longer lifetime, we have to copy or move the data from the ``sample``.
+Afterwards, we can start the applications which immediately connect to the RouDi via their runtime.
 
-This only allows us to get one sample at a time. Should we want to get all currently available samples we can do so
-by using an additional loop.
+When the application terminates, the runtime cleans up all resources needed for communication with RouDi. This
+includes all memory chunks used for the data transmission which may still be held by the application.
 
-```cpp
-while (keepRunning)
-{
-    while (subscriber.hasData())
-    {
-        subscriber->take()
-            .and_then([](iox::popo::Sample<const CounterTopic>& sample) {
-                CounterTopic* ptr = sample.get();
-                /* process the received data using the ptr */
-
-                /* alternatively use operator-> */
-                uint32_t counter = sample->counter;
-            })
-            .or_else([](iox::popo::ChunkReceiveResult) { /* handle the error */ });
-    }
-    // wait for new samples (either sleep or use the waitset)
-}
-```
 
-Here we do not check whether we actually have data since we already know there is data available by calling `hasData`.
+We now briefly define the main entities of an iceoryx system which were already used in the example above.
 
-### Untyped API
+## RouDi
 
-The untyped API offeres similar capabilities and is hence usable in a similar way. The major difference is that
-neither publisher nor subscriber have any knowledge about the underlying type they send or receive. This means that
-the user is responsible to ensure the data is read correctly, i.e. there is no type safety guaranteed by the API
-itself.
+RouDi is an abbreviation for **Rou**ting and **Di**scovery. RouDi takes care of the
+communication setup but does not actually participate in the communication between the publisher and the subscriber.
+RouDi can be thought of as the switchboard operator of iceoryx. One of its other major tasks is the setup of the
+shared memory, which the applications use for exchanging payload data. Sometimes referred to as daemon, RouDi manages
+the shared memory and is responsible for the service discovery, i.e. enabling subscribers to find topics offered by 
+publishers. It also keeps track of all applications which have initialized a runtime and are hence able to use
+publishers or subscribers. To view the available command line options call `$ICEORYX_ROOT/build/iox-roudi --help`.
 
```

Comment:
I think here we should add a section "Shared Memory". I suppose this will be a red box where the data packages aka topics will live inside, like on the current [GIF](https://user-images.githubusercontent.com/8661268/74612998-b962bc80-510a-11ea-97f0-62f41c5d287b.gif).

The section restrictions could be a sub-section of "Shared Memory"

Reply:
I wouldn't add this section here. I think many potential iceoryx users will know what Shared memory is and therefore a definition here is not needed. For those who are interested there is a link to our conceptual-guide.md at the beginning of this file.

Reply:
I would try to avoid to do such assumptions. We need to get every reader to the same level without prior knowledge. Users that already know what shared memory is, will just skip the section and jump to the next one. I'm happy to discuss the structure, though. However, shared memory is one of the basic elements of iceoryx, if not the main element. In my head it will be depicted by e.g. a red box in the graphic and therefore needs proper introduction.